profile/README.md

```diff
@@ -0,0 +1,41 @@
+![Foreman Header](https://raw.githubusercontent.com/theforeman/.github/main/profile/foreman-header.png)
+
+# The Foreman Project
+
+Foreman is a complete lifecycle management tool for physical and virtual servers. We give system administrators the power to easily automate repetitive tasks, quickly deploy applications, and proactively manage servers, on-premise or in the cloud.
+
+## Features
+
+### Provisioning
+
+Provision on bare-metal (MaaS) & public or private clouds all from one place with one simple process.
+
+### Configuration
+
+A complete configuration management solution including an ENC for Puppet and Salt, built-in support for parameterized classes and hierarchical parameter storage. Or integrate Ansible by assigning roles and variables.
```

Comment:
Would it make sense to link to change `ENC` into a link to explain what it is?
```suggestion
A complete configuration management solution including an [ENC](https://www.puppet.com/docs/puppet/8/nodes_external.html) for Puppet and Salt, built-in support for parameterized classes and hierarchical parameter storage. Or integrate Ansible by assigning roles and variables.
```

Reply:
A good suggestions, but as it is both Puppet and Salt I would suggest to use https://www.puppet.com/docs/puppet/latest/nodes_external.html for Puppet and https://docs.saltproject.io/en/latest/ref/tops/all/salt.tops.ext_nodes.html for Salt on the names. What do you think?

Reply:
I wasn't sure about Salt, but if they use the same name, then by all means link to both.